website/docs/conventions/git/branch-name.md

```diff
@@ -0,0 +1,46 @@
+---
+sidebar_label: Branch Name
+sidebar_position: 1
+---
+
+# Branch Naming Conventions
+
+To maintain consistency and clarity in our Git workflow, you could follow the
+format `<activity-type>/<activity-name>` to name a branch. The activity name
+should be **short, meaningful, and in plural form where applicable**, similar to
+how you would name a folder containing multiple related files.
+
+## Guidelines
+
+- **Keep branch names clean and focused**: do not include tracking system
+  activity IDs (e.g., Jira task IDs) in the branch name. Instead,
+  [reference them in the PR description](../pull-requests/format.md#description)
+- **Use lowercase and hyphens (`-`)** to separate words for readability
+- **Keep names concise but descriptive**, avoiding overly generic terms like
+  `update` or `fix`
+
+## Activity Types
+
+Use one of the following types to categorize the branch activity:
+
+- `feats`: for new features
```

Comment:
what about `docs`?

Reply:
@gunzip here are my thoughts:
- if the documentation is related to a `feature` or a `fix`, then it should be in the same branch of the activity
- if we are talking about a new guideline (like this one) or a new article, then I see those as new `features` (of the website application)
- if we need to fix some typos, formatting, or whatever to an existing document, then I'd see it as a `fix`

@gunzip do you see more cases where the `docs` category would fit better?

Reply:
I was thinking about the second point. Moreover, sometimes you just want to expand existing documentation. Would you call that a `fix`?

Reply:
it depends. I would probably use one of them because I can see that as a new feature (let's think about a new section of an existing guideline) or a fix (and I would see that as a fix to an existing page). I would evaluate at the moment, but right now I think I would use one of them.

Reply:
The main goal is to uniform the branches' names among teams. This could help also to have some sort of metrics, as you said.

Of course, you can have the same metrics on Jira, but I think there is a big assumption: every team uses the same task types and follows the same practices in handling the tasks. In this case it would be difficult to have meaningful stats

Reply:
I vote to add 'chores' and 'docs' categories to enable us to gather some insights about development investment: bugs vs features vs documentation vs technical debt. Let's see what @pagopa/engineering-team-devex think.